### PR TITLE
Update endpoint-dlp-learn-about.md

### PR DESCRIPTION
**Pull request by arunt14**
@chrfox , updated with the changes proposed. Please review

### CODE REVIEW

File: microsoft-365/compliance/endpoint-dlp-learn-about.md [hunk 1]
```diff
@@ -99,6 +99,10 @@ If you only want monitoring data from policy matches, you can turn off the **Alw
 > To ensure activities are audited for all supported file types, create a [custom DLP policy](create-test-tune-dlp-policy.md).
 
 Endpoint DLP monitors activity-based on MIME type, so activities will be captured even if the file extension is changed.
+The list of file types that are supported- 
+a) doc, docx, xls, xlsx, ppt, pptx, pdf : Supports after extension is changed to any file extension
+b) txt, pst, msg, rtf, c, cpp, h, cs, java, tsv : Supports if Extension is changed to only supported file extensions
+
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-learn-about.md]:
```suggestion

The list is a list of file types that are supported:

- doc, docx, xls, xlsx, ppt, pptx, pdf: Supports after the extension is changed to any file extension.
- txt, pst, msg, rtf, c, cpp, h, cs, java, tsv: Supports if the extension is changed to only supported file extensions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arunt14, kindly commit the suggested changes. Many thanks.

File: microsoft-365/compliance/endpoint-dlp-learn-about.md [hunk 2]
```diff
@@ -100,6 +100,11 @@ If you only want monitoring data from policy matches, you can turn off the **Alw
 
 Endpoint DLP monitors activity-based on MIME type, so activities will be captured even if the file extension is changed.
 
+The list is a list of file types that are supported:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-learn-about.md]:
```suggestion
The following is a list of file types that are supported:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @arunt14, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrfox** [comment]
back to arunt14 for completion of contributor license

**Comment by chrfox** [comment]
checking with PG to see if this is correct. also Contributor license agreement not signed

**Comment by arunt14** [comment]
@microsoft-github-policy-service agree [company="{Microsoft}"]

**Comment by arunt14** [comment]
@microsoft-github-policy-service agree

**Comment by arunt14** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by arunt14** [comment]
@chrfox , can you pls let me know when the changes will be published

**Pull request merged**